IYermakov/Snakes
# Reserve Dockerfile has been created

## Github Issue #

https://github.com/IYermakov/Snakes/issues/49

## What Are We Doing Here?

Reserve Dockerfile, which is based on original Tomcat image.

## How to Test and Verify

1. Check out this PR
2. To create image:  docker build -f webApp-based-on-Tomcat-Image/Dockerfile -t snakes:latest .
3. To run container locally: docker run -d -p 8080:8080 test-snakes

## Risk

### Level

`Low`
### Required Testing

`Smoke` `Manual`



## Code Review Checklist


we inevitably come back to this code it will only take hours to figure out, not
days.
